placement/algo/mirrored.go

```diff
@@ -192,6 +197,18 @@ func (a mirroredAlgorithm) ReplaceInstances(
 		return nil, err
 	}
 
+	// At this point, all leaving instances in the placement are cleaned up.
+	for i, instance := range addingInstances {
```

Comment:
This looks exactly the same as the previous block, refactor it into a common helper function?